include/LightGBM/c_api.h

```diff
@@ -988,6 +988,24 @@ LIGHTGBM_C_EXPORT int LGBM_BoosterFeatureImportance(BoosterHandle handle,
                                                     int importance_type,
                                                     double* out_results);
 
+/*!
+ * \brief Get model upper bound value.
+ * \param handle Handle of booster
+ * \param[out] out_results Result pointer to max value
```

Comment:
For the consistency with `LGBM_BoosterGetLowerBoundValue` function.
```suggestion
 * \param[out] out_results Result pointing to max value
```